.env.example

```diff
@@ -1,15 +1,13 @@
-# AMPLITUDE_WRITE_KEY='key_AMPLITUDE_WRITE_KEY'
-ATLASSIAN_CLIENT_ID='key_ATLASSIAN_CLIENT_ID'
-ATLASSIAN_CLIENT_SECRET='key_ATLASSIAN_CLIENT_SECRET'
-AUTH_INTERNAL_DISABLED='false'
-AUTH_GOOGLE_DISABLED='false'
-AUTH_SSO_DISABLED='false'
+# IMPORTANT
```

Comment:
+1 everything in here is important. Maybe call it server config or hosts or something like that?